qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -52,13 +55,49 @@ def del_users_and_groups(self):
                                        omit_sudo=False, check_status=False)
 
     def del_repos(self):
+        self.save_results_dir()
         self.mount_a.client_remote.run(args=f'sudo rm -rf {self.xfstests_repo_path}',
                                        omit_sudo=False, check_status=False)
 
         if self.install_xfsprogs:
             self.mount_a.client_remote.run(args=f'sudo rm -rf {self.xfsprogs_repo_path}',
                                            omit_sudo=False, check_status=False)
 
+    def save_results_dir(self):
+        """
+        When tests in xfstests-dev repo are executed, logs are created and
```

Comment:
This sounds reasonable. Could you check with teuthology folks if this is the recommended way?

Reply:
Recommended way for saving results dir? 

There's not instruction about AFAIK. I am doing this so that we have xfstests-dev's logs too when we run xfstests using teuthology.

Reply:
@zmc probably knows about this.